sv.c

```diff
@@ -16870,6 +16887,20 @@ S_find_uninit_var(pTHX_ const OP *const obase, const SV *const uninit_sv,
         }
         return varname(NULL, '$', obase->op_targ,
                        NULL, (I8)obase->op_private, FUV_SUBSCRIPT_ARRAY);
+
+    case OP_AELEMFASTLEX_STORE:
+        if (match) {
+            SV **svp;
```

Comment:
We're using C99 now, there's no need to declare `svp` at the beginning of the block.

Reply:
Hey, tell that to OP_AELEMFAST_LEX. I was only copying the existing code. :-)

Should I move the declarations in both places?

Reply:
Personally I dont think it matters much, that C99 allows us to do otherwise doesn't mean we should always do otherwise. We have some much older style code anyway it might be better to follow the existing style of the function.